src/display_network.rs

```diff
@@ -186,8 +186,8 @@ impl Network {
         );
         info_button.set_sensitive(false);
 
-        filter_button.connect_clicked(clone!(@weak filter_entry, @weak window => move |_| {
-            if filter_entry.get_visible() {
+        filter_button.connect_clicked(glib::clone!(@weak filter_entry, @weak window => move |_| {
+            if WidgetExt::is_visible(&filter_entry) {
```

Comment:
Why is it necessary to specify the trait here?

Reply:
I got this error in 4 different places (all for `is_visible`):

```
error[E0034]: multiple applicable items in scope
   --> src/display_network.rs:113:34
    |
113 |                 if !filter_entry.is_visible() || filter_entry.text_length() < 1 {
    |                                  ^^^^^^^^^^ multiple `is_visible` found
    |
    = note: candidate #1 is defined in an impl of the trait `CellRendererExt` for the type `O`
    = note: candidate #2 is defined in an impl of the trait `EntryExt` for the type `O`
    = note: candidate #3 is defined in an impl of the trait `TreeViewColumnExt` for the type `O`
    = note: candidate #4 is defined in an impl of the trait `WidgetExt` for the type `O`
help: disambiguate the associated function for candidate #1
    |
113 |                 if !CellRendererExt::is_visible(&filter_entry) || filter_entry.text_length() < 1 {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #2
    |
113 |                 if !EntryExt::is_visible(&filter_entry) || filter_entry.text_length() < 1 {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #3
    |
113 |                 if !TreeViewColumnExt::is_visible(&filter_entry) || filter_entry.text_length() < 1 {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: disambiguate the associated function for candidate #4
    |
113 |                 if !WidgetExt::is_visible(&filter_entry) || filter_entry.text_length() < 1 {
    |                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Too bad you only found that after the release :) This needs some renaming in the more specialized traits.

Reply:
I'll open an issue for it.